src/main/java/core/basesyntax/service/impl/TransactionParseServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionParseService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionParseServiceImpl implements TransactionParseService {
+    public static final int FIRST_LINE_INDEX = 0;
+    public static final String DEFAULT_TITLE = "type,fruit,quantity";
+    public static final int TYPE_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+    public static final String COMMA_SEPARATOR = ",";
```

Comment:
let's set proper access modifiers. Fix the issue with access modifiers in all places